tests/console_test.go

```diff
@@ -38,6 +38,24 @@ import (
 	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
+func withNodeSelectorLabel(label string, value string) libvmi.Option {
```

Comment:
Not sure about the name. `NodeSelector` would be `vmi.spec.NodeSelector`. This is `NodeAffinity`. See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity for more info.

Reply:
Personally, the `Label` part confuses me. Declaration as `func withNodeAffinityTo(label, valueOfLabel string) libvmi.Option` or similar seems more precise/